File: chapter2_reading_file.py
for line in open("emperors.txt"):
  line = line.split(",")
  name = line[0]
  birth_year = float( line[1] )
  death_year = float( line[2] )
  start_of_reign = float( line[3] )
  end_of_reign = float( line[4] )

File: chapter2_reading_file.r
data <- read.csv("emperors.txt", header = F)

for( i in 1:nrow(data) ){
  name <- data[i,1]
  birth_year <- data[i,2]
  death_year <- data[i,3]
  start_of_reign <- data[i,4]
  end_of_reign <- data[i,5]
}
